var/spack/repos/builtin/packages/py-neo/package.py

```diff
@@ -22,12 +25,18 @@ class PyNeo(PythonPackage):
     version("0.4.1", sha256="a5a4f3aa31654d52789f679717c9fb622ad4f59b56d227dca490357b9de0a1ce")
     version("0.3.3", sha256="6b80eb5bdc9eb4eca829f7464f861c5f1a3a6289559de037930d529bb3dddefb")
 
+    # py-setuptools@:61 doesn't support PEP 621
+    depends_on("py-setuptools@61:", type="build", when="@0.12.0:")
```

Comment:
```suggestion
    depends_on("py-setuptools@62:", type="build", when="@0.12:")
```
It's listed in pyproject.toml